src/index.umd.js

```diff
@@ -0,0 +1,84 @@
+export { Alert, InlineAlert } from './alert'
```

Comment:
Can we reuse the existing `index.js` export file? I realize you want to omit ssr, but if that's the only thing maybe we export everything from `index.js` and stub the ssr export with no-op functions for the umd bundle?

Reply:
You are right. I will try it. 👌

Reply:
At last I create `index.umd.js` in the `src/ssr` folder, make `extractStyles` and `autoHydrate` be an empty function, then replace the `./ssr` import string with `./ssr/index.umd.js` in the UMD build.

I think this can solve the reuse problem. If we have any changes in the future, we can keep the main entry unchanged.